src/pymor/models/transforms.py

```diff
@@ -0,0 +1,90 @@
+import numpy as np
+from scipy.linalg import null_space
+
+from pymor.core.base import ImmutableObject
+
+
+class MoebiusTransform(ImmutableObject):
+    """A Moebius transform operator.
+
+    Maps complex numbers to complex numbers.
+    The transform coefficients can be normalized.
+    Contains a method for constructing an inverse mapping, as well as a constructor that takes three
+    points and their images. The Moebius transforms form a group under composition, so __matmul__ is
+    defined to yield Moebius Transforms if both factors are Moebius Transforms.
+    """
+
+    def __init__(self, coefficients, normalize=False, name=None):
+        assert len(coefficients) == 4
+        coefficients = np.array(coefficients, dtype=complex)
+        assert coefficients[0]*coefficients[3] != coefficients[1]*coefficients[2]
+
+        if normalize:
+            coefficients /= np.sqrt(coefficients[0]*coefficients[3]-coefficients[2]*coefficients[3])
+
+        self.__auto_init(locals())
+
+    @classmethod
+    def from_points(cls, w, z=[0, 1, np.inf], name=None):
```

Comment:
It is good to avoid using mutable defaults.

```suggestion
    def from_points(cls, w, z=(0, 1, np.inf), name=None):
```